nvm.sh

```diff
@@ -1907,6 +1908,64 @@ nvm_download_artifact() {
   nvm_echo "${TARBALL}"
 }
 
+nvm_tar_knows_J() {
+  ERR_MSG=$(tar -J 2>&1 | head -1)
+  [ "_$ERR_MSG" != "_tar: unknown option -- J" ]
+}
+
+
+# args: tarball path, NVM_OS var
+nvm_extract_tar_artifact() {
+  local tar_compression_flag
+  local TARBALL
+  local TAR_COMMAND_SUFFIX
+  local TMPDIR
+  local NVM_OS
+  local tar
+
+  TARBALL="${1}"
+  NVM_OS="${2}"
+
+  tar='tar'
+  if [ "${NVM_OS}" = 'aix' ]; then
+    tar='gtar'
+  fi
+
+  tar_compression_flag='z'
+
+  if nvm_supports_xz "${VERSION}" && nvm_tar_knows_J; then
+    tar_compression_flag='J'
```

Comment:
let's move this `if` block into the `else` below, since that's the only path it matters.

Reply:
Ok, I'll try to upload it today.

Reply:
Done